src/frontend/templates/header.html

```diff
@@ -1,35 +1,78 @@
 {{ define "header" }}
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Hipster Shop</title>
-    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+    <title>Online Boutique Shop</title>
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB"
+        crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="/static/styles/styles.css">
+    <link rel="stylesheet" type="text/css" href="/static/styles/cart.css">
+    <link rel="stylesheet" type="text/css" href="/static/styles/order.css">
+    <link rel='shortcut icon' type='image/x-icon' href='/static/favicon.ico' />
 </head>
+
 <body>
 
     <header>
-        <div class="navbar navbar-dark bg-dark box-shadow">
+        <div class="navbar">
+            <div class="container d-flex justify-content-between">
+                <div class="h-free-shipping"><span>FREE</span> shipping with $75 purchase &nbsp;&nbsp;<span>see details</span></div>
+                <div class="h-controls">
+                    <div class="h-control">
+                        <img src="/static/icons/Hipster_SearchIcon.svg" alt="icon" class="icon search-icon" />
+                        <input type="text" placeholder="What are you looking for today?" />
+                    </div>
+                    <div class="h-control">
+                        <img src="/static/icons/Hipster_ProfileIcon.svg" alt="icon" class="icon" />
+                        <span>My Account</span>
+                    </div>
+                    <div class="h-control">
+                        <img src="/static/icons/Hipster_HelpIcon.svg" alt="icon" class="icon" />
+                        <span>Help</span>
+                    </div>
+                    <div class="h-control">
+                        <img src="/static/icons/Hipster_CurrencyIcon.svg" alt="icon" class="icon" />
+                        <select value="US Dollars">
+                            <option>US Dollars</option>
+                            <option>Euros</option>
+                            <option>CRC</option>
```

Comment:
To get this list (which was different) we made an RPC call, it would be so much better if we retain this part of the UI from Hipster Shop version IMO.

Reply:
currency conversion is not implemented in the onlineboutique frontend - working it add it now so that the list is actually populated.